src/gui/test/app/layout.tsx

```diff
@@ -0,0 +1,72 @@
+import { afterEach, test, vi, expect } from 'vitest'
+import { render, cleanup } from '@testing-library/react'
+import html from 'diffable-html'
+import { useGraphStore as useStore } from '@/state/index.js'
+import Layout from '@/layout.jsx'
+
+vi.mock('useGraphStore', () => ({
+  useGraphStore: vi.fn(),
+}))
+
```

Comment:
You shouldn't need to mock `useGraphStore`, you can refer to [selected-items tests](https://github.com/vltpkg/vltpkg/blob/main/src/gui/test/components/explorer-grid/selected-item.tsx) for a reference of how to test using the zustand store.